UnityProject/Assets/Scripts/Electricity/PowerSupplies/PowerGenerator.cs

```diff
@@ -8,16 +8,14 @@ public class PowerGenerator : NetworkBehaviour, ICheckedInteractable<HandApply>,
 {
 	[Tooltip("Whether this generator should start running when spawned.")]
 	[SerializeField]
-	private bool startAsOn = false;
+	private bool startAsOn;
```

Comment:
Produces a compiler warning. https://i.imgur.com/4YWm1cO.png.

Reply:
yeah, you need to give it a default value